packages/engine/lib/execution/src/package/simulation/state/behavior_execution/behavior/mod.rs

```diff
@@ -26,6 +26,12 @@ pub struct Behavior {
     pub behavior_keys_src: Option<String>,
 }
 
+impl Behavior {
+    pub fn is_python(&self) -> bool {
+        self.name.ends_with(".py")
```

Comment:
Not really happy about getting the language from the name but not asking to change this in this PR because this would be out of scope. However, could you use `Language::from_file_name` and return the language instead?

Reply:
Use `Language::from_file_name` in https://github.com/hashintel/hash/pull/546/commits/a17adc19a4b562c7bfcfd89335f7799ec251b57c and make the function return the language in https://github.com/hashintel/hash/pull/546/commits/aabc44f5e95078736bb6bc01aa0ad88f2491f6e1